azurerm/resource_arm_virtual_network_gateway_connection.go

```diff
@@ -125,6 +125,16 @@ func resourceArmVirtualNetworkGatewayConnection() *schema.Resource {
 				Computed: true,
 			},
 
+			"connection_protocol": {
```

Comment:
could we add an acceptance test covering this new field?

Reply:
will do

Reply:
added:
`$ make testacc TESTARGS='-run=TestAccAzureRMVirtualNetworkGatewayConnection_connectionprotocol'`
looong time after...
`--- PASS: TestAccAzureRMVirtualNetworkGatewayConnection_connectionprotocol (2512.56s)
PASS`